File: git-intro/fizzbuzz/c/fizzbuzz.c
#include <stdio.h>


int main(){

int lower = 25;
int upper = 150;

int fizz= 7;
int buzz = 12;
int is_fizz=0;
int is_buzz=0;
for(int i=lower; i<=upper;i++){
    is_fizz= (i%fizz==0);
      is_buzz= (i%buzz==0);
          

    

    if(is_fizz==0 && is_buzz==0){
        printf("\n %d",i);


    }
    if(is_fizz==1 && is_buzz==0){
        printf("\n fizz");


    }
        if(is_fizz==0 && is_buzz==1){
            printf("\n buzz",i);


    }
     if(is_fizz==1 && is_buzz==1){
            printf("\n fizzbuzz",i);


    }



}


}

File: git-intro/fizzbuzz/rust/fizzbuzz.rs
/*
    To run the code, you must have a rust installed
    Go to: https://www.rust-lang.org/tools/install
    Once installed, open your terminal and type:
        `rustc fizzbuzz.rs`
        `./fizzbuzz` - Linux or macOS
        `.\fizzbuzz.exe` - Windows

    ** Make sure you are in the rust folder so that the
    ** code file can be found.
*/

fn main() {
    // The range of numbers we wish to run the exercise on
    let lower: u8 = 1;
    let upper: u8 = 100;

    // The multiples we want print fizz and buzz on
    let fizz: u8 = 7;
    let buzz: u8 = 12;

    for i in lower..=upper {
        match (i % fizz, i % buzz) {
            (0, 0) => println!("fizzbuzz"),
            (0, _x) => println!("fizz"),
            (_x, 0) => println!("buzz"),
            _ => println!("{}", i),
        };
    }
}